special-interest-groups/README.md

```diff
@@ -0,0 +1,44 @@
+# OpenSearch Special Interest Groups ("SIGs")
+
+## Introduction
+
+This document defines the basic structure and life cycle of Special Interest Groups (SIGs) within the OpenSearch Project. More information about specific SIGs can be found within their respective charter documents.
+
+## Purpose
+
+An OpenSearch SIG will oversee and coordinate the interests pertaining to a logical area of needs of end users and/or projects. SIGs are:
```

Comment:
might be worth defining what logical area refers to. Taking examples is one route